src/main/java/frc/robot/subsystems/shooter/Aimer.java

```diff
@@ -54,18 +53,18 @@ public Aimer() {
         motorL.setSmartCurrentLimit(CONSTANTS.getNeo550BrushlessCurrentLimit());
         motorR.setSmartCurrentLimit(CONSTANTS.getNeo550BrushlessCurrentLimit());
         motorL.setSecondaryCurrentLimit(CONSTANTS.getNeo550BrushlessCurrentSecondaryLimit());
-        motorL.setSecondaryCurrentLimit(CONSTANTS.getNeo550BrushlessCurrentSecondaryLimit());
+        motorR.setSecondaryCurrentLimit(CONSTANTS.getNeo550BrushlessCurrentSecondaryLimit());
     }
 
     @Override
     public void periodic() {
-        // Log inputs
         updateInputs();
         Logger.processInputs("Shooter/Aimer", inputs);
 
         // Set Voltages
-        if (controller.getSetpoint() != 0) { // TODO: Determine Acceptable variance.
+        if (controller.getSetpoint() != 0) {
             double output = controller.calculate(inputs.currentAngleDegrees);
+            output = Math.min(output, 2);
```

Comment:
This limits the power in the positive direction; perhaps we should place a limit in the negative direction as well (clamp the output).